conf/README.md

```diff
@@ -0,0 +1,2 @@
+# Configuration
+This folder should contain `.hcl` files.
```

Comment:
### Note 1
We use the directory `conf/nomad/` for storing nomad_jobs.
Currently, I do not see other examples to use the `conf` directory, rather than storing nomad jobs in `conf/nomad/` dir.

### Note 2
I do not exclude the opportunity to have other types of files under `conf/` dir than `.hcl`

```suggestion
Directory `conf` uses for storing module configuration files.
```

Reply:
What other configuration files besides nomad jobs can be put here?

Reply:
Right now, I do not know. I just point here, that this folder may contain other types of files (not only .hcl)

Reply:
Ok, I will change to something similar to what you have suggested. If there's not any other configuration planned to be put in this folder I think it should be renamed to `nomad`. But that's another discussion.

Reply:
Yeap. I agree